README.md

```diff
@@ -14,174 +14,18 @@ Install this library using the python package manager:
-...     rsa_key = keyfile.read()
->>> credentials = PrivateCredentials(<consumer_key>, rsa_key)
->>> xero = Xero(credentials)
-```
-
-[Follow these steps](https://developer.xero.com/documentation/api-guides/create-publicprivate-key/) to generate a public/private key pair to sign your requests.  You'll upload your public key when you create your Xero Private app at https://app.xero.com.  You'll use the private key (aka RSA key) to generate your oAuth signature.
-
-The RSA key is a multi-line string that will look something like::
-
-    -----BEGIN RSA PRIVATE KEY-----
-    MIICXgIBAAKBgQDWJbmxJjQLGM76sZkk2EhsdpV0Gxtrhzh/wiNBGffa5JHV/Ex4
-    ....
-    mtXGQjKqsOpuCw7HwgnRQUWKYbaJ3a+yTCFjVwa9keQhDQ==
-    -----END RSA PRIVATE KEY-----
-
-You can get this string by either reading the contents of your private key
-file into a variable, or storing the key value as a constant. If you choose to
-store the key value as a constant, remember two things:
-
-* **DO NOT UNDER ANY CIRCUMSTANCES** check this file into a public
-  repository. It is your identity, and anyone with access to this file
-  could masquerade as you.
-
-* Make sure there is no leading space before
-  the ``-----BEGIN PRIVATE KEY-----`` portion of the string.
-
-
-### Partner Applications
-
-Partner Application authentication works similarly to the 3-step OAuth used by
-Public Applications, but with RSA signed requests. Partner OAuth tokens still
-have a 30 minute expiry, but can be swapped for a new token at any time.
-
-When you [register your partner application with Xero](https://developer.xero.com/documentation/auth-and-limits/partner-applications/), you'll have a **Consumer Key**, **Consumer Secret** and **RSA Key**. All three elements are required.
-
-
-```python
->>> from xero import Xero
->>> from xero.auth import PartnerCredentials
->>> credentials = PartnerCredentials(<consumer_key>, <consumer_secret>,
-...                                  <rsa_key>)
->>> xero = Xero(credentials)
-```
-
-When using the API over an extended period, you will need to exchange tokens
-when they expire.
-
-```python
->>> if credentials.expired():
-...     credentials.refresh()
-```
-
-**Important**: ``credentials.state`` changes after a token swap. Be sure to persist
-the new state.
-
 ### Using OAuth2 Credentials
 
-OAuth2 Credentials work similarly to the 3-step OAuth used by
-Partner Applications. OAuth2 tokens have a 30 minute expiry,
-but can be swapped for a new token at any time. Xero documentation on the OAuth2
-process can be found [here](https://developer.xero.com/documentation/oauth2/).
-The procedure for creating and authenticating credentials is as follows *(with a
-Django example at the end)*:
+*Xero have removed API access for "Public applications" and "Private
+applications" as of 31st March 2021 and 30th September 2021 respectively. From
+30th September 2021, "Partner applications" only support OAuth2 and are referred
+to as simply "OAuth2 apps".*
```

Comment:
Is this comment required? From a longevity point of view, in 2 years time, this line won't make sense to anybody; and if anyone has code that is using the Public/Private/Partner APIs, they are already broken, so they won't be looking for migration help. Is there anything to be gained by preserving references to the dead APIs?